### PR TITLE
update animals table

### PR DESCRIPTION
**Pull request by ybadr99**
- insert new data
- update some transactions
Screenshot from 2023-07-12 11-47-12
Screenshot from 2023-07-12 12-47-31

Screenshot from 2023-07-12 11-26-25

Screenshot from 2023-07-12 11-22-51

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -12,4 +12,33 @@ SELECT * FROM animals WHERE neutered IS TRUE;
...
+SAVEPOINT s1;
+UPDATE animals SET weight_kg = weight_kg*-1;
+ROLLBACK TO s1;
+UPDATE animals SET weight_kg = weight_kg*-1 WHERE weight_kg < 0;
+COMMIT;
+-- 4
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts >= 1;
+SELECT avg(weight_kg) FROM animals;
+SELECT max(escape_attempts) FROM animals;
```
Comment (ybadr99) [inline queries.sql]: I'm not sure if I understand it correct !

File: queries.sql [hunk 2]
```diff
@@ -12,4 +12,35 @@ SELECT * FROM animals WHERE neutered IS TRUE;
...
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-1-1';
+SAVEPOINT s1;
+UPDATE animals SET weight_kg = weight_kg*-1;
+ROLLBACK TO s1;
+UPDATE animals SET weight_kg = weight_kg*-1 WHERE weight_kg < 0;
+COMMIT;
+-- 4
+SELECT count(*) FROM animals;
+SELECT count(*) FROM animals WHERE escape_attempts >= 1;
```
Comment (TuyishimireEric) [inline queries.sql]: - [ ] Kindly the question is: `How many animals have never tried to escape?` please the query is not correct, Kindly fix this 🙏

**Review by TuyishimireEric — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :recycle: 
### Hi  @ybadr99   👋 👋 
#### I'm @TuyishimireEric, Your code reviewer,
Good job so far 👍 
 there are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights 

✔ Descriptive PR 👍 

## Required Changes ♻️

- [ ]  Kindly for the `schema.sql` it would be better to add a new column called `species` with a string type and the queries should be added to your file 🙏 

ex:
```sql
   ALTER TABLE animals ADD species varchar(255);
```

- Check the comments below 👇

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @ybadr99👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**